src/js/component/form.jsx

```diff
@@ -23,6 +29,13 @@ const uischema = {
     {
       type: "Control",
       scope: "#/properties/textControl",
+    },
+    {
+      type: "Control",
+      scope: "#/properties/multilineTextControl",
+      options: {
+        multi: true,
```

Comment:
`options` is part of uiSchema will help determine the renderer of the control
Question: can read the `rows` option out of the schema?